flink-jobs/src/main/java/com/github/projectflink/streaming/ThroughputMeasuringFlatMap.java

```diff
@@ -0,0 +1,72 @@
+package com.github.projectflink.streaming;
+
+import org.apache.flink.api.common.functions.FlatMapFunction;
+import org.apache.flink.api.java.utils.ParameterTool;
+import org.apache.flink.util.Collector;
+
+import com.github.projectflink.streaming.Throughput.Type;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetAddress;
+
+class ThroughputMeasuringFlatMap implements FlatMapFunction<Type, Integer> {
+	private static final Logger LOG = LoggerFactory.getLogger(ThroughputMeasuringFlatMap.class);
+
+	private static final long serialVersionUID = -4881110695631095859L;
+
+	private Integer host;
+	private long received;
+	private long start;
+	private int payload;
+	private long logfreq;
+	private long lastLog = -1;
+	private long lastElements;
```

Comment:
please initialize non-final members (as before) - that makes it a bit more explicit.